geocodeoffline/README.metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "category": "Search and Query",
+    "description": "Geocode addresses to locations and reverse geocode locations to addresses offline.",
+    "formal_name": "GeocodeOffline",
+    "ignore": false,
+    "images": [
+        "geocode-offline.png"
+    ],
+    "keywords": [
+        "geocode",
+        "geocoder",
+        "locator",
+        "offline",
+        "package",
+        "query",
+        "search",
+        "GeocodeParameters",
+        "GeocodeResult",
+        "LocatorTask",
+        "ReverseGeocodeParameters"
+    ],
+    "language": "kotlin",
+    "redirect_from": "",
+    "relevant_apis": [
+        "GeocodeParameters",
+        "GeocodeResult",
+        "LocatorTask",
+        "ReverseGeocodeParameters"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/arcgismaps/sample/geocodeoffline/DownloadActivity.kt",
+        "src/main/java/com/esri/arcgismaps/sample/geocodeoffline/MainActivity.kt"
```

```suggestion
        "src/main/java/com/esri/arcgismaps/sample/geocodeoffline/MainActivity.kt",
        "src/main/java/com/esri/arcgismaps/sample/geocodeoffline/DownloadActivity.kt"
```
Let's use MainActivity.kt as the first snippet so the [developer website](https://developers.arcgis.com/kotlin/sample-code/display-device-location-with-nmea-data-sources/#sample-code) displays the correct file first.